### ISSUE TITLE
Requiring "util" is not working with node

### ISSUE THREAD

**Issue by wclr** [garyb/purescript-debug#35]
When working with ES modules node doesn't support `require`, we don't have the ability to have an output using `util.inspect`. Not sure how to fix it best way, to have a separate module version for Node?

**Comment by garyb** [comment]
Hmm, yeah I think that'll have to be the case. Or maybe could add a node-specific module in here.

**Comment by wclr** [comment]
Yes, I mean `Debug.Node` module in this package. And it should be reimplemented in the same way as Affjax versions with passing specific output functions, not to duplicate the logic.
